src/components/squareComponent/square.spec.js

```diff
@@ -14,4 +14,55 @@ describe('Square Component', () => {
     ReactDOM.unmountComponentAtNode(gameSpaceDiv);
   });
 
+  it('tests the square is empty before the user has placed a marker on the square', () => {
+    const wrapper = mount(<GameSquare />);
+    expect(wrapper.state('valueX')).toBe('');
+    expect(wrapper.state('valueO')).toBe('');
+  });
+
+  it('handles placing an X on the square', () => {
+    const wrapper = mount(<GameSquare />);
+    const instance = wrapper.instance();
+    expect(wrapper.state('valueX')).toBe('');
+    instance.toggleX();
```

Comment:
Why are we both calling `toggleX` directly as well as simulating a `click` event? Shouldn't only one of the two be needed?

Also, in general, I would favor testing behavior over implementation, so I would lean towards testing the functionality via simulating the `click` event.

Reply:
I changed my code to reflect behavior testing via simulating the `click` event. 
(I'll have to do some more reading about implementation vs behavior testing)